easybuild/easyconfigs/m/MaxQuant/MaxQuant-2.2.0.0-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'Tarball'
+
+name = 'MaxQuant'
+version = '2.2.0.0'
+
+homepage = 'https://maxquant.org/maxquant'
+description = """MaxQuant is a quantitative proteomics software package designed for analyzing large
+ mass-spectrometric data sets. It is specifically aimed at high-resolution MS data. Several labeling
+ techniques as well as label-free quantification are supported."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+# requires registration at https://www.maxquant.org/download_asset/maxquant/latest
```

Comment:
again, use `download_instructions` since that will be more informative when the source tarball cannot be found